server.ts

```diff
@@ -0,0 +1,32 @@
+import { createServer } from 'http';
+import { parse } from 'url';
+import { readFileSync } from 'fs';
+// import { paths } from './src/utils/config';
+// import { join } from 'path';
+
+const port = process.env.PORT || 8080;
+// const eventsFile = join(paths['dist'], 'events.json');
+// const defaultFile = join(paths['root'], 'index.html');
+
+function initServer(req: any, res: any) {
+    const path = parse(req.url).pathname;
+    console.log('current path: ', path);
+
+    res.writeHead(200);
+    if (path == '/dist/events.json') {
```

Comment:
I’m surprised it needs a direct check like this. Is the server not able to just infer it’s a file? Admittedly I mostly work with express that abstracts a lot more of the lower level stuff.

Reply:
You're right actually, though we'd need middlewares and a few more installations and dependencies if we used express, so I thought we could go with the most basic bare-bones requirements for this one !